From 7b6a523d94fc51bbc316dd9db938d9453e4ad4ab Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Thu, 4 Jan 2018 14:41:49 +0200
Subject: [PATCH 3191/5242] MLK-17275-12: arm64: dts: fsl-imx8qm-mek: Remove
 dsi/lvds specific dts files

commit  5e59710ced9cfa953a92a17c45655b477c016542 from
https://source.codeaurora.org/external/imx/linux-imx.git

Delete the dst files specific to dsi/lvds nodes for the 8QM MEK platform.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/Makefile             |    4 -
 .../dts/freescale/fsl-imx8qm-mek-dsi-adv7535.dts   |  103 -----------
 .../freescale/fsl-imx8qm-mek-it6263-adv7535.dts    |  187 --------------------
 .../freescale/fsl-imx8qm-mek-it6263-rm67191.dts    |  173 ------------------
 .../boot/dts/freescale/fsl-imx8qm-mek-it6263.dts   |  120 -------------
 5 files changed, 587 deletions(-)
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dsi-adv7535.dts
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263-adv7535.dts
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263-rm67191.dts
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263.dts

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index e8d978d..fbfb8c6 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -22,12 +22,8 @@ dtb-$(CONFIG_ARCH_FSL_IMX8QM) += fsl-imx8qm-lpddr4-arm2.dtb \
 				 fsl-imx8qm-mek_ca53.dtb \
 				 fsl-imx8qm-mek_ca72.dtb \
 				 fsl-imx8qm-mek-hdmi.dtb \
-				 fsl-imx8qm-mek-it6263.dtb \
-				 fsl-imx8qm-mek-dsi-adv7535.dtb \
 				 fsl-imx8qm-mek-dsi-rm67191.dtb \
 				 fsl-imx8qm-mek-jdi-wuxga-lvds1-panel.dtb \
-				 fsl-imx8qm-mek-it6263-adv7535.dtb \
-				 fsl-imx8qm-mek-it6263-rm67191.dtb \
 				 fsl-imx8qm-lpddr4-arm2-dp.dtb \
 				 fsl-imx8qm-lpddr4-arm2-8cam.dtb \
 				 fsl-imx8qm-lpddr4-arm2-it6263.dtb \
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dsi-adv7535.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dsi-adv7535.dts
deleted file mode 100644
index ca003ad..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dsi-adv7535.dts
+++ /dev/null
@@ -1,103 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qm-mek.dts"
-
-&ldb1_phy {
-	status = "disabled";
-};
-
-&ldb1 {
-	status = "disabled";
-};
-
-&i2c0_mipi_dsi0 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_mipi0_lpi2c0>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	adv_bridge1: adv7535@3d {
-		compatible = "adi,adv7535", "adi,adv7533";
-		reg = <0x3d>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_1_in: endpoint {
-				remote-endpoint = <&mipi_dsi_bridge1_adv>;
-			};
-		};
-	};
-};
-
-&mipi_dsi_phy1 {
-	status = "okay";
-};
-
-&mipi_dsi1 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge1 {
-	status = "okay";
-
-	port@1 {
-		mipi_dsi_bridge1_adv: endpoint {
-			remote-endpoint = <&adv7535_1_in>;
-		};
-	};
-};
-
-&i2c0_mipi_dsi1 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_mipi1_lpi2c0>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	adv_bridge2: adv7535@3d {
-		compatible = "adi,adv7535", "adi,adv7533";
-		reg = <0x3d>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_2_in: endpoint {
-				remote-endpoint = <&mipi_dsi_bridge2_adv>;
-			};
-		};
-	};
-};
-
-&mipi_dsi_phy2 {
-	status = "okay";
-};
-
-&mipi_dsi2 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge2 {
-	status = "okay";
-
-	port@1 {
-		mipi_dsi_bridge2_adv: endpoint {
-			remote-endpoint = <&adv7535_2_in>;
-		};
-	};
-};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263-adv7535.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263-adv7535.dts
deleted file mode 100644
index ff1f0ff..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263-adv7535.dts
+++ /dev/null
@@ -1,187 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qm-mek.dts"
-
-&hdmi {
-	status = "disabled";
-};
-
-&ldb1_phy {
-	status = "okay";
-};
-
-&ldb1 {
-	status = "okay";
-
-	lvds-channel@0 {
-		fsl,data-mapping = "jeida";
-		fsl,data-width = <24>;
-		status = "okay";
-
-		port@1 {
-			reg = <1>;
-
-			lvds0_out: endpoint {
-				remote-endpoint = <&it6263_0_in>;
-			};
-		};
-	};
-};
-
-&i2c1_lvds0 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_lvds0_lpi2c1>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	lvds-to-hdmi-bridge@4c {
-		compatible = "ite,it6263";
-		reg = <0x4c>;
-
-		port {
-			it6263_0_in: endpoint {
-				clock-lanes = <3>;
-				data-lanes = <0 1 2 4>;
-				remote-endpoint = <&lvds0_out>;
-			};
-		};
-	};
-};
-
-&ldb2_phy {
-	status = "okay";
-};
-
-&ldb2 {
-	status = "okay";
-
-	lvds-channel@0 {
-		fsl,data-mapping = "jeida";
-		fsl,data-width = <24>;
-		status = "okay";
-
-		port@1 {
-			reg = <1>;
-
-			lvds1_out: endpoint {
-				remote-endpoint = <&it6263_1_in>;
-			};
-		};
-	};
-};
-
-&i2c1_lvds1 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_lvds1_lpi2c1>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	lvds-to-hdmi-bridge@4c {
-		compatible = "ite,it6263";
-		reg = <0x4c>;
-
-		port {
-			it6263_1_in: endpoint {
-				clock-lanes = <3>;
-				data-lanes = <0 1 2 4>;
-				remote-endpoint = <&lvds1_out>;
-			};
-		};
-	};
-};
-
-&i2c0_mipi_dsi0 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_mipi0_lpi2c0>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	adv_bridge1: adv7535@3d {
-		compatible = "adi,adv7535", "adi,adv7533";
-		reg = <0x3d>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_1_in: endpoint {
-				remote-endpoint = <&mipi_dsi_bridge1_adv>;
-			};
-		};
-	};
-};
-
-&mipi_dsi_phy1 {
-	status = "okay";
-};
-
-&mipi_dsi1 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge1 {
-	status = "okay";
-
-	port@1 {
-		mipi_dsi_bridge1_adv: endpoint {
-			remote-endpoint = <&adv7535_1_in>;
-		};
-	};
-};
-
-&i2c0_mipi_dsi1 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_mipi1_lpi2c0>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	adv_bridge2: adv7535@3d {
-		compatible = "adi,adv7535", "adi,adv7533";
-		reg = <0x3d>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_2_in: endpoint {
-				remote-endpoint = <&mipi_dsi_bridge2_adv>;
-			};
-		};
-	};
-};
-
-&mipi_dsi_phy2 {
-	status = "okay";
-};
-
-&mipi_dsi2 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge2 {
-	status = "okay";
-
-	port@1 {
-		mipi_dsi_bridge2_adv: endpoint {
-			remote-endpoint = <&adv7535_2_in>;
-		};
-	};
-};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263-rm67191.dts
deleted file mode 100644
index f347f54..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263-rm67191.dts
+++ /dev/null
@@ -1,173 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qm-mek.dts"
-
-&hdmi {
-	status = "disabled";
-};
-
-&ldb1_phy {
-	status = "okay";
-};
-
-&ldb1 {
-	status = "okay";
-
-	lvds-channel@0 {
-		fsl,data-mapping = "jeida";
-		fsl,data-width = <24>;
-		status = "okay";
-
-		port@1 {
-			reg = <1>;
-
-			lvds0_out: endpoint {
-				remote-endpoint = <&it6263_0_in>;
-			};
-		};
-	};
-};
-
-&i2c1_lvds0 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_lvds0_lpi2c1>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	lvds-to-hdmi-bridge@4c {
-		compatible = "ite,it6263";
-		reg = <0x4c>;
-
-		port {
-			it6263_0_in: endpoint {
-				clock-lanes = <3>;
-				data-lanes = <0 1 2 4>;
-				remote-endpoint = <&lvds0_out>;
-			};
-		};
-	};
-};
-
-&ldb2_phy {
-	status = "okay";
-};
-
-&ldb2 {
-	status = "okay";
-
-	lvds-channel@0 {
-		fsl,data-mapping = "jeida";
-		fsl,data-width = <24>;
-		status = "okay";
-
-		port@1 {
-			reg = <1>;
-
-			lvds1_out: endpoint {
-				remote-endpoint = <&it6263_1_in>;
-			};
-		};
-	};
-};
-
-&i2c1_lvds1 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_lvds1_lpi2c1>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	lvds-to-hdmi-bridge@4c {
-		compatible = "ite,it6263";
-		reg = <0x4c>;
-
-		port {
-			it6263_1_in: endpoint {
-				clock-lanes = <3>;
-				data-lanes = <0 1 2 4>;
-				remote-endpoint = <&lvds1_out>;
-			};
-		};
-	};
-};
-
-&mipi_dsi_phy1 {
-	status = "okay";
-};
-
-&mipi_dsi1 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge1 {
-	status = "okay";
-
-	panel@0 {
-		compatible = "raydium,rm67191";
-		reg = <0>;
-		pinctrl-0 = <&pinctrl_mipi_dsi_0_1_en>;
-		reset-gpio = <&gpio1 7 GPIO_ACTIVE_HIGH>;
-		dsi-lanes = <4>;
-		panel-width-mm = <68>;
-		panel-height-mm = <121>;
-		port {
-			panel1_in: endpoint {
-				remote-endpoint = <&mipi_bridge1_out>;
-			};
-		};
-	};
-
-	port@1 {
-		mipi_bridge1_out: endpoint {
-			remote-endpoint = <&panel1_in>;
-		};
-	};
-};
-
-&mipi_dsi_phy2 {
-	status = "okay";
-};
-
-&mipi_dsi2 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge2 {
-	status = "okay";
-
-	panel@0 {
-		compatible = "raydium,rm67191";
-		reg = <0>;
-		pinctrl-0 = <&pinctrl_mipi_dsi_0_1_en>;
-		reset-gpio = <&gpio1 7 GPIO_ACTIVE_HIGH>;
-		dsi-lanes = <4>;
-		panel-width-mm = <68>;
-		panel-height-mm = <121>;
-		port {
-			panel2_in: endpoint {
-				remote-endpoint = <&mipi_bridge2_out>;
-			};
-		};
-	};
-
-	port@1 {
-		mipi_bridge2_out: endpoint {
-			remote-endpoint = <&panel2_in>;
-		};
-	};
-};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263.dts
deleted file mode 100644
index d3dc1b0..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-it6263.dts
+++ /dev/null
@@ -1,120 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qm-mek.dts"
-
-&ldb1_phy {
-	status = "okay";
-};
-
-&ldb1 {
-	status = "okay";
-
-	lvds-channel@0 {
-		fsl,data-mapping = "jeida";
-		fsl,data-width = <24>;
-		status = "okay";
-
-		port@1 {
-			reg = <1>;
-
-			lvds0_out: endpoint {
-				remote-endpoint = <&it6263_0_in>;
-			};
-		};
-	};
-};
-
-&i2c1_lvds0 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_lvds0_lpi2c1>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	lvds-to-hdmi-bridge@4c {
-		compatible = "ite,it6263";
-		reg = <0x4c>;
-
-		port {
-			it6263_0_in: endpoint {
-				clock-lanes = <3>;
-				data-lanes = <0 1 2 4>;
-				remote-endpoint = <&lvds0_out>;
-			};
-		};
-	};
-};
-
-&iomuxc {
-	imx8qm-mek {
-		pinctrl_lvds0_lpi2c1: lvds0lpi2c1grp {
-			fsl,pins = <
-				SC_P_LVDS0_I2C1_SCL_LVDS0_I2C1_SCL	0xc600004c
-				SC_P_LVDS0_I2C1_SDA_LVDS0_I2C1_SDA	0xc600004c
-			>;
-		};
-		pinctrl_lvds1_lpi2c1: lvds1lpi2c1grp {
-			fsl,pins = <
-				SC_P_LVDS1_I2C1_SCL_LVDS1_I2C1_SCL	0xc600004c
-				SC_P_LVDS1_I2C1_SDA_LVDS1_I2C1_SDA	0xc600004c
-			>;
-		};
-	};
-};
-
-&ldb2_phy {
-	status = "okay";
-};
-
-&ldb2 {
-	status = "okay";
-
-	lvds-channel@0 {
-		fsl,data-mapping = "jeida";
-		fsl,data-width = <24>;
-		status = "okay";
-
-		port@1 {
-			reg = <1>;
-
-			lvds1_out: endpoint {
-				remote-endpoint = <&it6263_1_in>;
-			};
-		};
-	};
-};
-
-&i2c1_lvds1 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_lvds1_lpi2c1>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	lvds-to-hdmi-bridge@4c {
-		compatible = "ite,it6263";
-		reg = <0x4c>;
-
-		port {
-			it6263_1_in: endpoint {
-				clock-lanes = <3>;
-				data-lanes = <0 1 2 4>;
-				remote-endpoint = <&lvds1_out>;
-			};
-		};
-	};
-};
-- 
1.7.9.5

